packages/webapp/webapp_server.js

```diff
@@ -1205,6 +1208,25 @@ function runWebAppServer() {
         }
       }
 
+      let unixSocketGroup = process.env.UNIX_SOCKET_GROUP;
```

Comment:
I believe we don't need any let on this code. const + one new variable should be enough.

Reply:
If this is changed to const then the trim() operation on line 1213 would not work.

Reply:
Would you be happier with this?

`const unixSocketGroup = (env.UNIX_SOCKET_GROUP || "").trim();`

Reply:
I noticed that `whomst.sync.group()` can take both numerical gids and group names as an argument, so I was able to make the code more concise and eliminate all the lets, leaving one const.